spec/lib/email/styles_spec.rb

```diff
@@ -185,6 +185,14 @@ def html_fragment(html)
       styled = Nokogiri::HTML5.fragment(styled)
       expect(styled.at("test")["style"]).to eq("color:red;background:yellow")
     end
+    it "respects !important" do
```

Comment:
cvx-lint

```suggestion

    it "respects !important" do
```

Reply:
I was just matching the other tests in this block. But fine, I'll fix them all 😝